seatunnel-engine/seatunnel-engine-common/src/main/java/org/apache/seatunnel/engine/common/config/server/ServerConfigOptions.java

```diff
@@ -268,46 +52,304 @@ public class ServerConfigOptions {
+    /////////////////////////////////////////////////
 
-    public static final Option<Integer> PORT =
-            Options.key("port")
-                    .intType()
-                    .defaultValue(8080)
-                    .withDescription("The port of the http server.");
-
-    public static final Option<Boolean> ENABLE_HTTP =
-            Options.key("enable-http")
-                    .booleanType()
-                    .defaultValue(false)
-                    .withDescription("Whether to enable the http server.");
-
-    public static final Option<String> CONTEXT_PATH =
-            Options.key("context-path")
-                    .stringType()
-                    .defaultValue("")
-                    .withDescription("The context path of the http server.");
-
-    public static final Option<Boolean> ENABLE_DYNAMIC_PORT =
-            Options.key("enable-dynamic-port")
+    public static final Option<Boolean> TELEMETRY_LOGS_SCHEDULED_DELETION_ENABLE =
+            Options.key("scheduled-deletion-enable")
                     .booleanType()
-                    .defaultValue(false)
-                    .withDescription(
-                            "Whether to enable the dynamic port of the http server. If true, We will use the unused port");
-
-    public static final Option<Integer> PORT_RANGE =
-            Options.key("port-range")
-                    .intType()
-                    .defaultValue(100)
+                    .defaultValue(true)
                     .withDescription(
-                            "The port range of the http server. If enable-dynamic-port is true, We will use the unused port in the range");
+                            "Enable scheduled cleanup, with default value of true. The system will automatically delete relevant log files when job expiration time, as defined by `history-job-expire-minutes`, is reached. "
+                                    + "If this feature is disabled, logs will remain permanently on disk, requiring manual management, which may affect disk space usage. It is recommended to configure this setting based on specific needs.");
 
-    public static final Option<HttpConfig> HTTP =
-            Options.key("http")
-                    .type(new TypeReference<HttpConfig>() {})
-                    .defaultValue(new HttpConfig())
-                    .withDescription("The http configuration.");
+    public static final Option<TelemetryLogsConfig> TELEMETRY_LOGS =
+            Options.key("logs")
+                    .type(new TypeReference<TelemetryLogsConfig>() {})
+                    .defaultValue(new TelemetryLogsConfig())
+                    .withDescription("The telemetry logs configuration.");
 
-    public static final String EVENT_REPORT_HTTP = "event-report-http";
-    public static final String EVENT_REPORT_HTTP_URL = "url";
-    public static final String EVENT_REPORT_HTTP_HEADERS = "headers";
+    public static class MasterServerConfigOptions {
```

Comment:
Re-struct this class. 
Split the options to `MasterServerConfigOptions` and `SlaveServerConfigOptions`